packages/analytics/src/helpers.ts

```diff
@@ -95,7 +92,9 @@ export function getOrCreateDataLayer(dataLayerName: string): DataLayer {
  */
 function wrapGtag(
```

Comment:
This function is getting pretty big and I found it a little hard to follow. An idea is to break it into 3 functions, one for each command type.

Also I think it will make the code more readable if you use await async instead of chaining promises. The change in return type should not matter as the caller can choose to ignore it.

Reply:
Done.